ntbs-service/Helpers/ExtensionMethods.cs

```diff
@@ -10,4 +12,16 @@ public static class DateTimeExtension
             return new FormattedDate() { Day = dateTime?.Day.ToString(), Month = dateTime?.Month.ToString(), Year = dateTime?.Year.ToString() };
         }
     }
+
+    public static class EnumExtension
+    {
+        public static TAttribute GetAttribute<TAttribute>(Enum enumValue) 

```

Comment:
if you make it a `public static TAttribute GetAttribute<TAttribute>(this Enum enumValue)` (note the use of `this`), then you have a much more fluent way of calling it:
`ClinicalDetails.HIVTestState.GetAttribute<DisplayAttribute>().Name`
But also, since we're making it more generically accessible, perhaps we need some checks to make sure it behaves nicely... let's grab a few minutes to iron this one out? :)